src/Views/View.js

```diff
@@ -1,12 +1,14 @@
-import {sources,settings} from "../../src/js/constant/constant";
+import {sources,settings} from "../js/constant/constant";
 import "../js/component/BootstrapMenu";
-class Views {
+class View {
   constructor(sources, settings) {
     this.sources = sources;
     this.settings = settings;
     this.errorsModal = document.querySelector(settings.errorsModal)
     this.main = document.querySelector(settings.gridSection)
     this.nav = document.querySelector(settings.navSection)
+    this.ul = document.querySelector(settings.ulSelector)
+    this.channelUILists = document.querySelector(settings.channelUIListsSelector)
     this.errorsModal = document.querySelector(settings.errorsModal)
```

Comment:
do we need the second definition of this.errorModal?

Reply:
Updated - Only single definition Now